py/DREAM/Settings/Equations/RunawayElectrons.py

```diff
@@ -101,6 +118,51 @@ def setInitialProfile(self, density, radius=0):
         self.verifySettingsPrescribedInitialData()
 
 
+    def setHottailFinalTemperature(self, T_final, radius=0):
+        _data, _rad = self._setInitialData(data=T_final, radius=radius)
+
+        self.hottail_T_final = _data
+        self.hottail_T_final_r  = _rad
+        self.verifySettingsPrescribedInitialData()
+        
+    
+    # Loss term    
+    def setLCFSLoss(self, lcfs_loss):
+        """
+        Specifies which model to use for calculating the
+        LCFS loss term.
+        """
+        if lcfs_loss == False:
+            self.lcfs_loss = LCFS_LOSS_MODE_DISABLED
+        else:
+            self.lcfs_loss = int(lcfs_loss)
+    
+    
+    # Loss term
+    def setLCFSLossTime(self, t_loss, radius=0):
+        """
+        Sets the timescale constant t_loss for the
+        LCFS loss term. 
+        """
+        _data, _rad = self._setInitialData(data=t_loss, radius=radius)
+
+        self.lcfs_t_loss = _data
+        self.lcfs_t_loss_r  = _rad
+        self.verifySettingsPrescribedInitialData()
+        
+        
+    def setLCFSLossPsiEdget0(self, psi_edge_t0, user_input_active=1):
+        """
+        Sets the value of psi_p at the plasma edge at
+        t = 0, used to determine the LCFS radial point.
+        Use in restarts to keep the value from the first
+        simulation. user_input_active should be 1 (default)
+        or set to 0 to manually switch off the user input.
+        """
+        self.lcfs_user_input_psi = int(user_input_active)
```

Comment:
Yeah, here it really looks like ``user_input_active`` should be a boolean instead!